lib/ansible/modules/cloud/azure/azure_rm_storageblob.py

```diff
@@ -81,7 +81,7 @@
     resource_group:
         description:
             - Name of the resource group to use.
-        required: true
+            - If C(resource_group) is not provided, the field C(shared_access_key) must be provided.
```

Comment:
Required when I(shared_access_key) not defined.

Reply:
https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-within-module-documentation

Reply:
done